docs/serving/services/deployment.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Deployment Configuration"
+linkTitle: "Deployment Configuration"
+weight: 03
+type: "docs"
+---
+
+The Deployment ConfigMap contains settings that configure how Kubernetes Deployment resource, backing Knative Services, is configured.
+
+## Accessing the Deployment ConfigMap 
+
+The Deployment ConfigMap is named `config-deployment` and is located in the `knative-serving` namespace.
```

Comment:
Remove this line considering above comment? :slightly_smiling_face: just a suggestion for readability.